inbox/models/folder.py

```diff
@@ -37,8 +37,18 @@ class Folder(MailSyncBase, UpdatedAtMixin, DeletedAtMixin):
     # folders as per
     # https://msdn.microsoft.com/en-us/library/ee624913(v=exchg.80).aspx
     name = Column(CategoryNameString(), nullable=False)
-    canonical_name = Column(String(MAX_INDEXABLE_LENGTH), nullable=False,
-                            default='')
+    _canonical_name = Column(String(MAX_INDEXABLE_LENGTH), nullable=False,
+                             default='', name="canonical_name")
+
+    @property
+    def canonical_name(self):
+        return self._canonical_name
+
+    @canonical_name.setter
+    def canonical_name(self, value):
```

Comment:
I think the ideal way to do the default value would be `value=''` in the function declaration!